src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/RecalibrationTablesMerger.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.cloud.dataflow.sdk.coders.DefaultCoder;
+import com.google.cloud.dataflow.sdk.coders.SerializableCoder;
+import com.google.cloud.dataflow.sdk.transforms.Combine;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationTables;
+
+import java.io.Serializable;
+
+/**
+ * Dataflow-ese for "aggregate RecalibrationTables using their 'combine' method".
+ * <p>
+ * See https://cloud.google.com/dataflow/java-sdk/JavaDoc/com/google/cloud/dataflow/sdk/transforms/Combine.CombineFn
+ * for the Dataflow spec this follows. I've copied their javadoc here.
+ */
+@DefaultCoder(SerializableCoder.class)
+public class RecalibrationTablesMerger extends Combine.CombineFn<RecalibrationTables, RecalibrationTablesMerger.Acc, RecalibrationTables> implements Serializable {
+
+    @DefaultCoder(SerializableCoder.class)
+    public class Acc implements Serializable {
```

Comment:
do we need this class? can you use RecalibrationTables directly? ALso, note that NestedIntegerArray is not serializable right now